website/src/pages/accelerate-dev-onboarding.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FunctionComponent, useEffect } from 'react'
+import { Link, PageProps } from 'gatsby'
+
+import Layout from '../components/Layout'
+import { createHubSpotForm } from '../components/HubSpot'
+
+const FindAndFixVulnerabilities: FunctionComponent<PageProps> = props => {
```

Comment:
Can we perhaps rename this? This and the other page live at the top level under different URLs, so perhaps it would make sense to name them accordingly? Or, if we are thinking of them as two versions of the same thing, perhaps something like `SocialPage`?